packages/inputs/src/props.ts

```diff
@@ -479,7 +488,7 @@ export interface FormKitInputSlots<Props extends FormKitInputs<Props>> {
   file: FormKitFileSlots<Props>
   checkbox: Props['options'] extends AllReals
     ? FormKitBoxSlots<Props>
-    : FormKitBaseSlots<Props>
+    : FormKitCheckboxSlots<Props>
```

Comment:
Hmm, I believe this *only* includes the decorators instead of extending the base slots.

Reply:
ah right, my bad ✋ fixed it now 👍